components/x-gift-article/src/GiftArticle.jsx

```diff
@@ -226,19 +236,25 @@ const withGiftFormActions = withActions(
 						showHighlightsSuccessMessage: false
 					}
 				}
+			},
+			initOShare(selector) {
```

Comment:
**Question**: Are you sure a nested function like this is what's needed?

Reply:
alternatively, we could replace calling `initOShare` with initialising oShare directly, seeing as we only call it in [one place](https://github.com/Financial-Times/x-dash/blob/f271cf211dc80148a3923a5f835188a8abd12887/components/x-gift-article/src/v2/CreateLinkButton.jsx#L18). 

See [dfc7aae](https://github.com/Financial-Times/x-dash/pull/713/commits/dfc7aae2d0a24f2f55cd8871369624541060f66d)